### PR TITLE
Mark versionbot as Deprecated in action.yml

### PR DESCRIPTION
**Pull request by maggie0002**
Change-type: minor

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ inputs:
     required: false
     default: balena-cloud.com
   versionbot:
-    description: Does the repository use VersionBot ?
+		description: (Deprecated) Does the repository use VersionBot?
```
Comment (ab77) [inline action.yml:22]: spacing

**Review by ab77 — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (maggie0002) [reply action.yml]: Some weird code editor behaviour going on, keeps reverting back to tabs instead of spaces

File: action.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ inputs:
     required: false
     default: balena-cloud.com
   versionbot:
-    description: Does the repository use VersionBot ?
+    description: (Deprecated) Does the repository use VersionBot?
```
Comment (ab77) [inline action.yml:22]: @maggie0002 adding these brackets could break the action.

**Review by ab77 — CHANGES_REQUESTED** [review]

File: action.yml (hunk 2 above)
Comment (maggie0002) [reply action.yml:22]: It seems very unlikely in yaml like this, but its GitHub actions, so I wouldn't bet the bank on it.

`description: Deprecated - Does the repository use VersionBot?`?

**Review by ab77 — APPROVED** [review]

**Pull request merged**